# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/ciphers/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -pruN ciphers-v3.0a_orig/Makefile ciphers-v3.0a/Makefile
--- ciphers-v3.0a_orig/Makefile	2004-11-18 17:51:13.000000000 +0100
+++ ciphers-v3.0a/Makefile	2005-01-11 12:31:41.261344216 +0100
@@ -150,7 +150,7 @@ endif
 loop_twofish.$(VM1)o: loop_twofish.c
 	$(CP1) $(CC) $(CFLAGS) $(EF) -DKBUILD_BASENAME=loop_twofish -DKBUILD_MODNAME=loop_twofish -c $(PP1)loop_twofish.c -o $(PP1)loop_twofish.o
 ifneq "$(VM1)" ""
-	$(CP1) $(MP1) vmlinux $(PP1)loop_twofish.o $(VM3) >/dev/null 2>&1
+	$(CP1) $(MP1) $(PP1)loop_twofish.o $(VM3)
 	$(CP1) $(CC) $(CFLAGS) $(EF) -DKBUILD_BASENAME=loop_twofish_mod -DKBUILD_MODNAME=loop_twofish -c $(PP1)loop_twofish.mod.c -o $(PP1)loop_twofish.mod.o
 	$(LD) $(LF) loop_twofish.o loop_twofish.mod.o -o loop_twofish.$(VM1)o
 	rm -f loop_twofish.o loop_twofish.mod.[co] $(VM3).mod.[co]
@@ -159,7 +159,7 @@ endif
 loop_serpent.$(VM1)o: loop_serpent.c
 	$(CP1) $(CC) $(CFLAGS) $(EF) -DKBUILD_BASENAME=loop_serpent -DKBUILD_MODNAME=loop_serpent -c $(PP1)loop_serpent.c -o $(PP1)loop_serpent.o
 ifneq "$(VM1)" ""
-	$(CP1) $(MP1) vmlinux $(PP1)loop_serpent.o $(VM3) >/dev/null 2>&1
+	$(CP1) $(MP1) $(PP1)loop_serpent.o $(VM3)
 	$(CP1) $(CC) $(CFLAGS) $(EF) -DKBUILD_BASENAME=loop_serpent_mod -DKBUILD_MODNAME=loop_serpent -c $(PP1)loop_serpent.mod.c -o $(PP1)loop_serpent.mod.o
 	$(LD) $(LF) loop_serpent.o loop_serpent.mod.o -o loop_serpent.$(VM1)o
 	rm -f loop_serpent.o loop_serpent.mod.[co] $(VM3).mod.[co]
@@ -168,7 +168,7 @@ endif
 loop_blowfish.$(VM1)o: loop_blowfish.c
 	$(CP1) $(CC) $(CFLAGS) $(EF) -DKBUILD_BASENAME=loop_blowfish -DKBUILD_MODNAME=loop_blowfish -c $(PP1)loop_blowfish.c -o $(PP1)loop_blowfish.o
 ifneq "$(VM1)" ""
-	$(CP1) $(MP1) vmlinux $(PP1)loop_blowfish.o $(VM3) >/dev/null 2>&1
+	$(CP1) $(MP1) $(PP1)loop_blowfish.o $(VM3)
 	$(CP1) $(CC) $(CFLAGS) $(EF) -DKBUILD_BASENAME=loop_blowfish_mod -DKBUILD_MODNAME=loop_blowfish -c $(PP1)loop_blowfish.mod.c -o $(PP1)loop_blowfish.mod.o
 	$(LD) $(LF) loop_blowfish.o loop_blowfish.mod.o -o loop_blowfish.$(VM1)o
 	rm -f loop_blowfish.o loop_blowfish.mod.[co] $(VM3).mod.[co]
